### PR TITLE
Fix show[Structure] & show[Syntax] number literals suffixes (#459)

### PR DESCRIPTION
**Pull request by xavier-fernandez**
- Long literal suffixes will always be an uppercase ’L’
- Float literal suffixes will always be a lowercase ’f’
- Double literal suffixes will always be a lowercase ’d’

### CODE REVIEW

**Comment by xavier-fernandez** [comment]
Signed the CLA

File: scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeStructure.scala [hunk 1]
```diff
@@ -56,7 +56,13 @@ object TreeStructure {
             case Ident("-") => true
             case _ => false
           }
-          s(x.tokens.filter(isRelevantToken _).map(_.show[Syntax]).mkString)
+          def showToken(tok: Token) = tok match {
+            case Constant.Long(v) => Show.Long(v.longValue)
+            case Constant.Float(v) => Show.Float(v.floatValue)
+            case Constant.Double(v) => Show.Double(v.doubleValue)
```
Comment (xeno-by) [inline scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeStructure.scala]: I suggest we convert these constants to `Show.Str` in order to keep the `Show` framework as simple as possible.

Comment (xavier-fernandez) [inline scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeStructure.scala]: Ok, will fix this tomorrow morning :)

**Comment by xavier-fernandez** [comment]
Did a second commit with the @xeno-by feedback. If the second commit suits better, I will squash both of them :)

**Comment by xeno-by** [comment]
@xavier-fernandez Thanks! Please squash, and I'll merge.

**Comment by xavier-fernandez** [comment]
@xeno-by Both commits are squashed :)

**Comment by xeno-by** [comment]
LGTM

**Pull request merged**